js/packages/core-base/src/locate-text.ts

```diff
@@ -18,8 +18,21 @@ export function makeLocateText({requests, logger: defaultLogger}: Options) {
     settings: LocateTextSettings<TPattern>
     logger?: Logger
   }): Promise<LocateTextResult<TPattern>> {
+    settings.normalization ??= {}
+    settings.normalization.limit = {
+      maxImageHeight: Math.min(
+        settings.normalization.limit?.maxImageHeight ?? Infinity,
+        requests.test.account.maxImageHeight,
+      ),
+      maxImageArea: Math.min(
+        settings.normalization.limit?.maxImageArea ?? Infinity,
+        requests.test.account.maxImageArea,
+      ),
+    }
```

Comment:
Since `locate`, `extractText` and `locateText` are part of the `Core` now, you don't have a test started there. The way to get account info will be:
```suggestion
    const account = await requests.getAccountInfo({settings, logger})
    settings.normalization ??= {}
    settings.normalization.limit = {
      maxImageHeight: Math.min(
        settings.normalization.limit?.maxImageHeight ?? Infinity,
        account.maxImageHeight,
      ),
      maxImageArea: Math.min(
        settings.normalization.limit?.maxImageArea ?? Infinity,
        account.maxImageArea,
      ),
    }
```